package.json

```diff
@@ -12,7 +12,7 @@
     "lint": "next lint",
     "test": "jest --coverage",
     "type-check": "tsc --noEmit",
-    "format": "prettier --check .",
+    "format": "prettier --check --write .",
```

Comment:
@jadzlnds can you please revert this change?

`npm run format` is executed in the CI/CD pipeline. Changing the file content on the fly will mascarade issues.